src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -1455,6 +1459,101 @@ static void triangles_command(std::string masterIP, int connFd, SQLiteDBInterfac
     }
 }
 
+static void streaming_triangles_command(std::string masterIP, int connFd, JobScheduler* jobScheduler,
+                                        bool *loop_exit_p, int numberOfPartitions) {
+    int uniqueId = JasmineGraphFrontEnd::getUid();
+    int result_wr = write(connFd, GRAPHID_SEND.c_str(), FRONTEND_COMMAND_LENGTH);
+    if (result_wr < 0) {
+        frontend_logger.error("Error writing to socket");
+        *loop_exit_p = true;
+        return;
+    }
+    result_wr = write(connFd, "\r\n", 2);
+    if (result_wr < 0) {
+        frontend_logger.error("Error writing to socket");
+        *loop_exit_p = true;
+        return;
+    }
+
+    // We get the name and the path to graph as a pair separated by |.
+    char graph_id_data[301];
+    bzero(graph_id_data, 301);
+
+    read(connFd, graph_id_data, 300);
+
+    string graph_id(graph_id_data);
+    graph_id.erase(std::remove(graph_id.begin(), graph_id.end(), '\n'), graph_id.end());
+    graph_id.erase(std::remove(graph_id.begin(), graph_id.end(), '\r'), graph_id.end());
```

Comment:
Lets delete the above two line and replace that with `Utils::trim_copy(graphID, " \f\n\r\t\v");`

Reply:
`graph_id = Utils::trim_copy(graphID, " \f\n\r\t\v");`

Reply:
Done

Reply:
Ok